tsl/test/expected/continuous_aggs-13.out

```diff
@@ -2210,8 +2210,10 @@ AS
 SELECT time_bucket('1day', timec), min(location), sum(temperature), sum(humidity)
 FROM conditions
 GROUP BY time_bucket('1day', timec) WITH NO DATA;
+-- enable partial aggregates only for postgres version <= 14
+SELECT CASE WHEN current_setting('server_version_num')::int/10000 <= 14 THEN 'false' ELSE 'true' END AS "final_flag" \gset
```

Comment:
I guess this is not required anymore since you added the check for DEBUG build flag, or am I missing something?

Reply:
Added only to tests -ve testcase on PG15.